src/qt/guiutil.cpp

```diff
@@ -243,6 +243,7 @@ QString HtmlEscape(const QString& str, bool fMultiLine)
     if(fMultiLine)
     {
         escaped = escaped.replace("\n", "<br>\n");
+        escaped = escaped.replace(" ", "&nbsp;");
```

Comment:
Related: I suppose that changing the spaces to non-breaking ones cause problems when pasting the output into a json file and trying to parse it with a json parser? E.g. according to [RFC4627](https://www.ietf.org/rfc/rfc4627.txt), `%xa0` is not a valid whitespace character:
```
      ws = *(
                %x20 /              ; Space
                %x09 /              ; Horizontal tab
                %x0A /              ; Line feed or New line
                %x0D                ; Carriage return
            )
```